microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -40,6 +40,7 @@ class LtpTestsuite(TestSuite):
         timeout=_TIME_OUT,
         requirement=simple_requirement(
             min_core_count=8,
+            min_memory_mb=28 * 1024,
```

Comment:
Remove this requirement, since it's not needed now.

Reply:
It may cause other problems, like failed on small disk sizes.

Reply:
removed